blog/Migration-to-Fable2.md

```diff
@@ -0,0 +1,160 @@
+</div>
+</div>
+
+## Upgrade to Babel 7
+
+During Fable 2, development [Babel 7](https://babeljs.io/blog/2018/08/27/7.0.0) has been released so we need to upgrade to it. Hopefully for us, they created a tool to help us.
+
+1. Install [npx](https://github.com/zkat/npx#readme), this a CLI program allowing us to run npm packages.
+
+    - For npm: `npm install -g npx`
+    - For yarn: `yarn global add npx`
+
+2. Run `npx babel-upgrade --write`, this will modify the nearest `package.json` and convert all you old `babel-` reference to their scoped reference if their exist.
+
+    For example, `babel-polyfill` is converted to `@babel/polyfill`
+
+## Upgrade fable's npm package
+
+1. You need to upgrade `fable-loader` to latest version `2.x.x`
+
+    - For npm: `npm update fable-loader@latest`
+    - For yarn: `yarn upgrade fable-loader@latest`
+
+2. We don't need anymore `fable-utils`, so remove it
+
+    - For npm: `npm uninstal fable-utils`
+    - For yarn: `yarn remove fable-utils`
+
+
+## Update your webpack.config.js
+
+<div class="message is-info">
+<div class="message-body">
+
+In order to best benefit, of new Fable 2 tree-shaking and module size reduction we encourage you to upgrade all your webpack related package to their latest version.
+
+</div>
+</div>
+
+Here are the key points to consider for updating your `webpack.config.js`:
+- Custom `resolve.modules` isn't needed anymore. Now, fable will create a `.fable` folder located near your `package.json` so JS tools works natively
+    You can remove this lines from your `webpack.config.js`
+    ```js
+    resolve: {
+        modules: [
+            "node_modules/",
+            resolve("./node_modules/")
+        ]
+    },
+    ```
+- Remove any reference to `fable-utils` like:
+    - `const fableUtils = require("fable-utils");`
+    - `fableUtils.resolveBabelOptions`
+- Convert your `babelOptions` and any thing related to `babel-` to use `@babel` scope if available
+
+    **babel options**
+    ```js
+    var babelOptions = fableUtils.resolveBabelOptions({
+        presets: [
+            ["env", {
+                "targets": {
+                    "browsers": ["last 2 versions"]
+                },
+                "modules": false
+            }]
+        ]
+    });
+    ```
+    becomes
+    ```js
+    var babelOptions = {
+        presets: [
+            ["@babel/preset-env", {
+                "targets": {
+                    "browsers": ["last 2 versions"]
+                },
+                "modules": false
+            }]
+        ]
+    };
+    ```
+
+    **babel polyfill**
+
+    "babel-polyfill", becomes "@babel/polyfill"
+
+- Because Fable 2, now create a `.fable` folder you are not forced anymore to use **absolute** filepath. All the filepath are now relative to the nearest `package.json` found by Fable. When running Fable, it will tell you what it's *current working directory* is.
```

Comment:
Because Fable 2 now `creates` a `.fable` folder you are not forced anymore to use **absolute** `filepaths`. All the `filepaths` are now relative to the nearest `package.json` found by Fable. When running Fable, it will tell you what `its` *current working directory* is.